### ISSUE TITLE
Starting to work on something with similar goals

### ISSUE THREAD

**Issue by jefflarkin** [andyleap/indieserv#1]
Hi @andyleap. I wasn't sure how else to reach out to you, so I thought an issue might work. I attempted to leave you an #indieweb comment from my withknown site (https://jefflarkin.withknown.com/), but either my end messed up or your end isn't ready for web replies. I too was thinking of writing an indieweb-enabled CMS in go. Where are you hosting your site? I thought about writing something on app-engine, since I've not used it before and have been curious about its usability. Any chance you've tried your software there?

I don't have a ton of go experience, but that's kinda the point of writing the CMS myself. It's a toss-up to me between Go and Node, but since you've recently started something I thought I might look at what you've started and see if there's anything I can contribute.

**Comment by andyleap** [comment]
I'm on the indieweb irc under the alias Vendan, just fyi.  And yeah, there seems to be an issue with webmentions.  Still tracking it down, though real life gets in the way at times.  Not sure I'd recommend working with indieserv itself.  I have made a collection of utility libraries that would be quite handy, if you are interested.  Feel free to fork and submit pull requests though :D.

Most stuff should work fine on GAE, as I'm not using the unsafe package and such, but I do host on my own VPS, so I can't promise anything in that regard.

**Comment by jefflarkin** [comment]
Yes, as I've been looking through your code I've noticed that you've modularized a lot of it into other libraries, which I agree is likely a better way to go. Anyway, I'm hoping to do some programming on vacation next week, so I wanted to reach out in case I do. Hopefully I can re-use a lot of what you've written and contribute back some pull requests. I'm still trying to decide whether I want to use Node or Go, but right now I'm working on getting enough foundation written in both that I can see which I find more enjoyable to write. =) Thanks for commenting back.
